README.md

```diff
@@ -354,17 +357,24 @@ gitlab_letsencrypt: no
 # gitlab_ssl_crt: some_file.crt
 # # If you'd like to use letsencrypt, use this scructure.
 # gitlab_letsencrypt: yes
-# gitlab_letsencrypt_contact_emails:
-#   - robert@meinit.nl
-# gitlab_letsencrypt_group: root
-# gitlab_letsencrypt_key_size: 2048
-# gitlab_letsencrypt_owner: root
-# gitlab_letsencrypt_wwwroot: /var/opt/gitlab/nginx/www
-# gitlab_letsencrypt_auto_renew: yes
-# gitlab_letsencrypt_auto_renew_hour: 0
-# gitlab_letsencrypt_auto_renew_minute: nil
-# gitlab_letsencrypt_auto_renew_day_of_month: nil
-# gitlab_letsencrypy_auto_renew_log_directory: /var/log/gitlab/lets-encrypt
+gitlab_letsencrypt_contact_emails:
+  - bulumaknight@gmail.com
+gitlab_letsencrypt_group: root
+gitlab_letsencrypt_key_size: 2048
+gitlab_letsencrypt_owner: root
+gitlab_letsencrypt_wwwroot: /var/opt/gitlab/nginx/www
+gitlab_letsencrypt_auto_renew: yes
+gitlab_letsencrypt_auto_renew_hour: 0
+gitlab_letsencrypt_auto_renew_minute: nil
+gitlab_letsencrypt_auto_renew_day_of_month: nil
```

Comment:
cronjob failed adding: Failed to parse int from nil

https://github.com/robertdebock/ansible-role-gitlab/issues/2

Reply:
could you check, just pushed an update f577f2ef9551ea0740dc1c444a498b6d07283893